src/pandroid/app/src/main/java/com/panda3ds/pandroid/data/config/GlobalConfig.java

```diff
@@ -0,0 +1,61 @@
+package com.panda3ds.pandroid.data.config;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.panda3ds.pandroid.app.PandroidApplication;
+import com.panda3ds.pandroid.utils.Constants;
+
+import java.io.Serializable;
+
+public class GlobalConfig {
+    private static SharedPreferences data;
+
+    public static void initialize() {
+        data = PandroidApplication.getAppContext()
+                .getSharedPreferences(Constants.PREF_GLOBAL_CONFIG, Context.MODE_PRIVATE);
+    }
+
+    private static <T extends Serializable> T get(Key<T> key) {
```

Comment:
I might be mistaken, but shouldn't these get/set static methods be public? So they can be accessed from other classes? Or do we not need that?

Reply:
True, this really should be public